tests/experimental/evals/functions/test_binary.py

```diff
@@ -212,7 +211,8 @@ def test_run_relevance_eval(
             },
             status=200,
         )
-    model = OpenAIModel()
+    with patch.object(OpenAIModel, "_init_tiktoken", return_value=None):
```

Comment:
Could use a fixture so this doesn't need to be repeated? https://github.com/Arize-ai/phoenix/blob/main/tests/conftest.py